src/nvim/option.h

```diff
@@ -91,15 +91,14 @@ enum {
   SOPT_GLOBAL = 0x08,  ///< Option has global value
   SOPT_WIN    = 0x10,  ///< Option has window-local value
   SOPT_BUF    = 0x20,  ///< Option has buffer-local value
-  SOPT_UNSET  = 0x40,  ///< Option does not have local value set
 };
 
-/// Option types for various functions in option.c
-enum {
+/// Requested option scopes for various functions in option.c
+typedef enum {
   SREQ_GLOBAL = 0,  ///< Request global option value
```

Comment:
with typedef:ed enums we also use a style something like `kOptReqGlobal` , `kOptReqWin` etc for the constants.